packages/solid-query/src/createBaseQuery.ts

```diff
@@ -41,30 +41,22 @@ export function createBaseQuery<
     observer.getOptimisticResult(defaultedOptions),
   )
 
-  const [dataResource, { refetch, mutate }] = createResource<TData | undefined>(
-    () => {
-      return new Promise((resolve) => {
-        if (!(state.isFetching && state.isLoading)) {
-          resolve(unwrap(state.data))
-        }
-      })
-    },
-  )
-
-  batch(() => {
-    mutate(() => unwrap(state.data))
-    refetch()
+  let res = noop
```

Comment:
Can we give this variable a more descriptive name? Maybe `resolveQueryData` would be nice? 
```suggestion
  let resolveQueryData = noop
```